cli/commands.go

```diff
@@ -28,6 +28,8 @@ const (
 	defaultServerCount = 1
 )
 
+var defaultBindMounts = []string{"/dev/mapper:/dev/mapper"}
```

Comment:
Maybe this shouldn't be in `defaultBindMounts`, since it's not default, but only added in specific cases, right?

Reply:
sortof, it checks if it exists so it's a defaultBindMount but only applied when it exists. Could change that naming, but not sure what that'd get us since it's still sorta a default.